spec/linter-codeclimate-spec.js

```diff
@@ -35,4 +37,21 @@ describe('The codeclimate provider for Linter', () => {
       ),
     ),
   );
+
+  it('runs all configured engines', () =>
```

Comment:
This description is _badly_ misleading. All that this is testing is that it handles a single engine on a single file type...